source/UniverseObjects.h

```diff
@@ -132,6 +132,7 @@ class UniverseObjects {
 	std::map<const Sprite *, std::string> landingMessages;
 	std::map<const Sprite *, double> solarPower;
 	std::map<const Sprite *, double> solarWind;
+	std::map<const Sprite *, std::string> starIcon;
```

Comment:
Like leaf said, this should be a Sprite pointer and not a string. No use in passing strings around everywhere to then just get the Spriter pointer in the end anyways.